File: 백준/Bronze/2765. 자전거 속도/자전거 속도.cc
#define _USE_MATH_DEFINES
#include <cmath>
#include <iostream>
using namespace std;
int main(void) {
    double l, r, t, dist, mph, pi = M_PI;
    int cnt = 1;
    while (true) {
        cin >> l >> r >> t;
        if (r == 0) break;
        dist = pi * l * r / 12 / 5280;
        mph = dist / (t / 3600);
        cout << fixed;
        cout.precision(2);
        cout << "Trip #" << cnt++ << ": " << dist << ' ' << mph << "\n";
    }
    return 0;
}

File: 백준/Bronze/2765. 자전거 속도/자전거 속도.py
import sys,math
input=lambda:sys.stdin.readline().rstrip()
cnt=0
while True:
    l,r,t=map(float,input().split())
    if r==0: break
    dist=math.pi*l*r/12/5280
    mph=dist/(t/3600)
    cnt+=1
    print(f"Trip #{cnt}: {round(dist,2):.2f} {round(mph,2):.2f}")